beets/util/artresizer.py

```diff
@@ -127,9 +129,11 @@ def im_resize(maxwidth, path_in, path_out=None, quality=0, max_filesize=0):
     # "-resize WIDTHx>" shrinks images with the width larger
     # than the given width while maintaining the aspect ratio
     # with regards to the height.
+    # no interlace seems to be default, specify for explicitness
```

Comment:
Again, a full sentence here (with capital letter and period) might be a nice state to leave this in.

Reply:
fixed